data/bingo/functions/tick/tick.mcfunction

```diff
@@ -52,7 +52,7 @@ execute as @a[scores={bingo.resources=91}] run function bingo:util/go_to_lobby
 
 # Place the sign in resource pack check chamber
 setblock 1 2 1 minecraft:air
-setblock 1 2 1 minecraft:warped_wall_sign{Text1: '{"translate": "%1074992263$s%1$s", "color": "#8eedeb", "with": ["", {"translate": "%1074992263$s%3$s%1$s", "with": ["", {"translate": "bingo.resourcepack_check.sign.line1"}]}], "clickEvent": {"action": "run_command", "value": "/trigger bingo.resources"}}', Text2: '{"translate": "%1074992263$s%1$s", "color": "#8eedeb", "with": [{"text": "Download", "bold":true}, {"translate": "%1074992263$s%3$s%1$s", "with": [{"text": "Download", "bold":true}, {"translate": "bingo.resourcepack_check.sign.line2"}]}]}', Text3: '{"translate": "%1074992263$s%1$s", "color": "#8eedeb", "with": [{"text": "Resourcepack", "bold":true}, {"translate": "%1074992263$s%3$s%1$s", "with": [{"text": "Resourcepack", "bold":true}, {"translate": "bingo.resourcepack_check.sign.line3"}]}]}', Text4: '{"translate": "%1074992263$s%1$s", "color": "#8eedeb", "with": ["", {"translate": "%1074992263$s%3$s%1$s", "with": ["", {"translate": "bingo.resourcepack_check.sign.line4"}]}]}'}
+setblock 1 2 1 minecraft:warped_wall_sign{Text1: '{"translate": "%1074992263$s%1$s", "color": "#8eedeb", "with": ["", {"translate": "%1074992263$s%3$s%1$s", "with": [{"translate": "%1074992263$s%3$s%3$s%1$s", "with": ["", {"translate": "bingo.resourcepack_check.wrong_version.sign.line1"}], "bold":true}, {"translate": "bingo.resourcepack_check.sign.line1"}]}], "clickEvent": {"action": "run_command", "value": "/trigger bingo.resources"}}', Text2: '{"translate": "%1074992263$s%1$s", "color": "#8eedeb", "with": [{"text": "Download", "bold":true}, {"translate": "%1074992263$s%3$s%1$s", "with": [{"translate": "%1074992263$s%3$s%3$s%1$s", "with": [{"text": "Download"}, {"translate": "bingo.resourcepack_check.wrong_version.sign.line2"}], "bold":true}, {"translate": "bingo.resourcepack_check.sign.line2"}]}]}', Text3: '{"translate": "%1074992263$s%1$s", "color": "#8eedeb", "with": [{"text": "Resourcepack", "bold":true}, {"translate": "%1074992263$s%3$s%1$s", "with": [{"translate": "%1074992263$s%3$s%3$s%1$s", "with": [{"text": "Resourcepack"}, {"translate": "bingo.resourcepack_check.wrong_version.sign.line3"}], "bold":true}, {"translate": "bingo.resourcepack_check.sign.line3"}]}]}', Text4: '{"translate": "%1074992263$s%1$s", "color": "#8eedeb", "with": ["", {"translate": "%1074992263$s%3$s%1$s", "with": [{"translate": "%1074992263$s%3$s%3$s%1$s", "with": ["", {"translate": "bingo.resourcepack_check.wrong_version.sign.line4"}], "bold":true}, {"translate": "bingo.resourcepack_check.sign.line4"}]}]}'}
```

Comment:
Unminified Text2 as example:
```json
{
	"translate": "%1074992263$s%1$s",
	"color": "#8eedeb",
	"with": [
		{
			"text": "Download",
			"bold": true
		},
		{
			"translate": "%1074992263$s%3$s%1$s",
			"with": [
				{
					"translate": "%1074992263$s%3$s%3$s%1$s",
					"with": [
						{ "text": "Download" },
						{
							"translate": "bingo.resourcepack_check.wrong_version.sign.line2"
						}
					],
					"bold": true
				},
				{ "translate": "bingo.resourcepack_check.sign.line2" }
			]
		}
	]
}
```
I don't really see why you are still checking `%1074992263$s%1$s` here.
Wouldn't this have the same effect (also simplified `{ "text": "Download" }`

```json

{
	"translate": "%1074992263$s%3$s%1$s",
	"color": "#8eedeb",
	"with": [
		{
			"translate": "%1074992263$s%3$s%3$s%1$s",
			"with": [
				"Download",
				{
					"translate": "bingo.resourcepack_check.wrong_version.sign.line2"
				}
			],
			"bold": true
		},
		{ "translate": "bingo.resourcepack_check.sign.line2" }
	]
}
```

Reply:
Left the `<bingo><1>` (`%1074992263$s%1$s`) in case we wanted different messages for `no Bingo RP` and `wrong RP version`

Reply:
Hmm, it's not exactly hard to add it again later. I like to keep stuff clean when there are no actual plans to use it 🤷‍♂️

Reply:
Sure ;)

Reply:
Done